q2_phylogeny/_align_to_tree_mafft_raxml.py

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2019, QIIME 2 development team.
```

Comment:
Sorry @mikerobeson, missed this:

```suggestion
# Copyright (c) 2016-2020, QIIME 2 development team.
```

(this is what [travis is complaining about right now](https://travis-ci.org/qiime2/q2-phylogeny/builds/643535846#L841-L842))

Reply:
Yarp, I just noticed that as I committed the changes. Forgot to merge in the prior updates. :-/